private/pkg/storage/matcher.go

```diff
@@ -45,25 +52,19 @@ func MatchPathBase(base string) Matcher {
 
 // MatchPathEqual returns a Matcher for the path.
 func MatchPathEqual(equalPath string) Matcher {
-	return pathMatcherFunc(func(path string) bool {
-		return path == equalPath
```

Comment:
On windows should this be [`strings.EqualFold`](https://pkg.go.dev/strings#EqualFold)?